package.json

```diff
@@ -4,12 +4,13 @@
   "description": "Open source UI components and visual style guide for U.S. government websites",
   "main": "src/js/start.js",
   "scripts": {
-    "prepublish": "gulp build",
+    "prepublish": "bundle && gulp build",
```

Comment:
I personally think that we shouldn't require ruby as a dependency to work on the project and we should just put `gulp no-test build` here, but am fine with this for now.